src/modules/confirmation/components/ConfirmationSendTransactionDetails.vue

```diff
@@ -117,7 +120,11 @@ export default {
       ).value;
     },
     summaryItems() {
-      return ['Transaction Fee', 'Total'];
+      const defaultArr = ['Transaction Fee'];
```

Comment:
Could also be `return this.isNetworkCurrency ? ['Transaction Fee', 'Total'] : ['Transaction Fee']`